src/mousetracker.js

```diff
@@ -1400,11 +1400,20 @@
          * @function
          */
         removeContact: function() {
-            --this.contacts;
-
-            if (this.contacts < 0) {
-                $.console.warn('GesturePointList.removeContact() Implausible contacts value');
-                this.contacts = 0;
+            if (
+                (this.type === "mouse" ||
+                    this.type === "pen" ||
+                    this.type === "touch") &&
+                this.contacts > 0
+            ) {
+                --this.contacts;
+
+                if (this.contacts < 0) {
```

Comment:
With the new `contacts > 0` test, this won't ever happen, so we might as well remove it. 

On the other hand, maybe we shouldn't add the `contacts > 0` test, and instead rely on this, which has the extra feature of setting contacts to 0 if somehow they get into the negative range.

Reply:
The if statement was moved from outside the function. I did not dare to remove it. But I totally agree on your comment

Reply:
I'm concerned about losing the "setting contacts to 0 if somehow they get into the negative range" feature... Ideally, it's not necessary, but I'm not confident in saying it will never happen. I'm okay with removing the console warning, though.